lib/scutil/pgnewfil.c

```diff
@@ -19,9 +19,10 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 
-#if defined(HOST_WIN)
+#ifdef _WIN64
 #include <direct.h>
 #include <io.h>
+#include <sys/stat.h>
```

Comment:
Why does `sys/stat.h` need to be included again? It is already included above on line 19.